### PR TITLE
Chore/opt in telemetry

### PR DESCRIPTION
**Pull request by ivanThePleasant**
### What does it do?

This PR re-enables Strapi telemetry in case it was disabled. This PR introduces a CLI command that acts opposite to the `telemetry:disable` command that has been added recently.

### Why is it needed?

Gives an ability to re-enable telemetry easily should the user decide to do so.This is another step towards transparency of how our telemetry system works.

### How to test it?

Clone the repo, run `yarn strapi telemetry:enable` from the `getstarted` project. This will alter the `package.json` file.

### CODE REVIEW

**Review by fdel-car — APPROVED** [review]
I'm eager to start using node 14 as a minimum version to have nullish coalescing operator and optional chaining 😅

Everything looks to match what we had for the opt-out command, I was wondering couldn't we have a toggle telemetry command instead?
Or maybe even simpler for our users, just a set telemetry state function and call it from those two commands, just to avoid code duplication?

Otherwise, LGTM :)

**Comment by ivanThePleasant** [comment]
Yep, I agree on optional chaining 😅

As to the toggle command: when I was writing the script, I remember I had a very compelling argument as to why I wanted to have them split. But I can't remember what it was anymore 😅 Still, I think that keeping it separate for now is not an issue. We can always merge it later and make a toggle command if that's going to be a better strategy.

I guess I'd like to see one more approve before merging

**Comment by derrickmehaffy** [comment]
I believe we need documentation for this before we merge also unless we already have a docs PR?

File: packages/core/strapi/lib/commands/opt-in-telemetry.js
Comment (ivanThePleasant) [reply packages/core/strapi/lib/commands/opt-in-telemetry.js:80]: Correct, but that will also mean user did not use the disable command before, hence I didn't add the check here. You think it should be there as well?

File: packages/core/strapi/lib/commands/opt-in-telemetry.js
Comment (ivanThePleasant) [reply packages/core/strapi/lib/commands/opt-in-telemetry.js:91]: Actually, yes, thanks for noticing this. Makes more sense for it to be there 👍

**Comment by ivanThePleasant** [comment]
@derrickmehaffy yep, I will ask Shaun to change docs once this PR is reviewed and polished. Won't merge it until docs are ready :)

File: packages/core/strapi/lib/commands/opt-in-telemetry.js
Comment (alexandrebodin) [reply packages/core/strapi/lib/commands/opt-in-telemetry.js:85]: I see your point 🤔 I would maybe log it's already enabled and still write to the file to make it more explicit. WDYT ?

**Review by alexandrebodin — APPROVED** [review]
LGTM.

**Pull request merged**
